17 modernizr update

jhogue opened the pull request:
Relates to Issue #17 

Add our most recent version of the touch-hover mixin to the scaffold. Includes a set of variables for the Modernizr hook. Authors can change the hook in one place if they are using a newer version of modernize. 

Old Modernizr: `no-touch / touch`
New Modernizr: `no-touchevents / touchevents`

bendoh commented:
Looks nice, well-commented, etc. /approved

raubin commented on base/_typesizes.scss:
```diff
@@ -63,6 +63,18 @@ $type-sizes: (
+// and return with a specific element at a particular breakpoint
+// @jhogue
+//
+// @usage: .bravo { font-size: type-size(h2,large); }
```
@jhogue - isn't this backwards to how you've defined this function?

jhogue replied:
Hang on, thats not a file that I changed in this branch. Let me investigate the source and get back to you.

jhogue commented:
@kbeck303 @nicholemaureen @johncionci @BrandonDH @syzygy333 

Team… Cionci recently requested a fallback for when Modernizr is not available. I added that as a variable that we can switch true/false depending on the theme, and that lets us continue to write and structure our code in the same way every time. 

I also added documentation for preferred use case, how to use the `disabled` option, and what output to expect. 

Please take a look and lets get this into Master soon so John can use the code for Washfunders.

jhogue commented:
This will close Issue #17. The variables can be changed depending on the project version of Modernizr.

johncionci commented:
@jhogue  Looks good from here, nice cleanup and alpha order updates. Did we test? if not, I can pull into WASH tomorrow and try it out.

@kbeck303 @nicholemaureen @BrandonDH @syzygy333
Thoughts?

jhogue commented:
I tested only by running the code is Sassmeister to work out the kinks and get the sample output. I know it will work and will not produce an error, but I'd like to see you use it in the field to determine if we need to change anything.

kbeck303 commented on base/_typesizes.scss:
```diff
@@ -93,6 +106,14 @@ $type-sizes: (
+//	h3 { font-size: 1.3125rem; }
+//	h4 { font-size: 1.1875rem; }
+//	p { font-size: 1.0625rem; }
+//
```
needs the closing media bracket }

kbeck303 commented on base/_mixins.scss:
```diff
@@ -2,35 +2,106 @@
+		&,
+		&:link,
+		&:visited {
+		  @content;
```
the indents of `@content;` throughout this mixin are inconsistent

kbeck303 replied:
also tabs versus spaces in general throughout as well

kbeck303 commented:
I really like the "usage" and "return" examples for the mixins, that is super helpful (in my opinion).

jhogue commented:
@kbeck303 I fixed that up, thanks for noticing.

bendoh commented:
Still looks fine /approved
